code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
 import data from './data.json'
+import Header from './Header'
+import { Albumcontainer } from './Albumcontainer'
 
-console.log(data)
-
+// const artistsWithComma = artists.join(', ');
+// const artistsarray = artists.map(e => e.name).join(' ');
 export const App = () => {
+//   const dataartist = data.items.artists.name;
+// const artistcomma = dataartist.map(item=>item.join(', '));
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <> 
+    <div className="headermain">
+      < Header /></div>
```

Comment:
Do you need this `div` in the `App` or would it make sense to move it to the `<Header/>` component?